src/core/vector/qgsvectorlayer.h

```diff
@@ -2937,6 +2939,8 @@ class CORE_EXPORT QgsVectorLayer : public QgsMapLayer, public QgsExpressionConte
     //! Pointer to data provider derived from the abastract base class QgsDataProvider
     QgsVectorDataProvider *mDataProvider = nullptr;
 
+    QgsVectorLayerSelectionProperties *mSelectionProperties = nullptr;
```

Comment:
Even though it's in the private section, just for the sake of consistency, maybe add a //! comment to match what's done with the pointer above and below this?

Reply:
I'll just remove the other redundant comments -- they aren't adding any value